src/elements/content-preview/preview-header/Header.scss

```diff
@@ -32,7 +53,7 @@
     }
 
     .bp-base-header {
-        padding: 0 0 0 15px;
+        padding: 0 15px;
```

Comment:
Where was the right 15px padding accounted for previously?

Reply:
It wasn't, which was causing the button and title to be spaced differently, which is what this change aims to resolve.